libs/extensions/std/lang/src/example-validation.spec.ts

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  JayveeServices,
+  ValidationResult,
+  createJayveeServices,
+  readJvTestAssetHelper,
+  useExtension,
+  validationHelper,
+} from '@jvalue/jayvee-language-server';
+import { AstNode } from 'langium';
+import { NodeFileSystem } from 'langium/node';
+
+import { StdLangExtension } from './extension';
+
+describe('jv example tests', () => {
+  let services: JayveeServices;
+  let validate: (input: string) => Promise<ValidationResult<AstNode>>;
+
+  const readJvTestAsset = readJvTestAssetHelper(
+    __dirname,
+    '../../../../../example/',
+  );
+
+  beforeAll(() => {
+    // Register std extension
+    useExtension(new StdLangExtension());
+    // Create language services
+    services = createJayveeServices(NodeFileSystem).Jayvee;
+    // Create validation helper for language services
+    validate = validationHelper(services);
+  });
+
+  it.each([
+    'cars.jv',
+    'gas.jv',
```

Comment:
Note that the `gas.jv` file was replaced by `electric-vehicles.jv` in the main branch.

Reply:
Yeah I will have to merge soon, but thanks for the heads up 👍